.github/workflows/continous-integration.yaml

```diff
@@ -33,18 +33,32 @@ jobs:
       - name: Lint python
         run: poetry run flake8 **/*.py
 
-      - name: Install Terraform
```

Comment:
This is hanging due to a bug, so I've commented it out until https://jira.digital.nhs.uk/browse/APM-608 is resolved